__tests__/components/__snapshots__/LessonCard.js.snap

```diff
@@ -47,6 +47,7 @@ exports[`Lesson Card Complete State Should render lessonCard with loading... 1`]
             class="lesson-card__title font-weight-bold mt-3"
           >
             <a
+              class=" "
```

Comment:
why is there a `class=" "`? Shouldn't it have no class?

Reply:
Right, i was debated whether or not to fix this. This is because of the way I am rendering the NavLinks, would it be best to add more conditionals for when no className is passed in?